python/py-hggit/Portfile

```diff
@@ -6,10 +6,10 @@ PortGroup               bitbucket 1.0
 
 epoch                   20130201
 
-bitbucket.setup         durin42 hg-git 8d00fde45adbc6c3c0ccab8e362b5f5c36c171e6
+bitbucket.setup         durin42 hg-git c8fea7a921634bda3d4e68e2c7a0d73b7447dea4
 name                    py-hggit
-version                 0.8.12
-revision                1
+version                 20190402
```

Comment:
To answer your question: if you want to go back to 0.8.13, you need to increase epoch.

Reply:
ok. I don't like to set a fake 0.8.13-alpha or something similar so increment epoch seems the only way

Reply:
Anyway, this port already has the epoch set, so it won't be any different if you just increase it.
Still, version `20190402` is not official either.